.travis/custom_release.sh

```diff
@@ -3,10 +3,20 @@
 NODE_ENV=production npm run build
 
 if [ "${TRAVIS_BRANCH}" = "master" ]; then
+    echo "PUSHING ci-beta"
+    rm -rf dist/.git
     .travis/release.sh "ci-beta"
+
+    echo "PUSHING qa-beta"
+    rm -rf dist/.git
```

Comment:
Should we not just use a for loop?
```
for env in ci qa
do
  echo "PUSHING ${env}-beta"
  rm -rf dist/.git
  .travis/release.sh "${env}-beta"
done

Reply:
I was thinking about it then wanted to make the smallest change, but agree for consistency it makes sense to use make it look like the other repo scripts.  I'll update.